addons/account/static/tests/tours/tax_group_tests.js

```diff
@@ -124,3 +124,33 @@ registry.category("web_tour.tours").add('account_tax_group', {
         trigger: '.o_tax_group_amount_value:contains("120")',
     },
 ]});
+
+registry.category("web_tour.tours").add('search_read_tour', {
```

Comment:
Can you move it in a new JS file ?
A bit weird here since it's not related to any tax group